DateTimes/DateTimes/Program.cs

```diff
@@ -0,0 +1,426 @@
+            //  1h)  
+            Console.Write("1h) The day of week is: ");
+            Console.WriteLine($"Today is day no. {DayOfTheWeek()} of the {WeekOfTheYear()}");
+            // 1j)
+            Console.WriteLine(IsLeapYear());
+            Console.WriteLine("*************************************************************************************");
+        }
+        #endregion
+
+        #region 01d Write a program to display week number of the year
+        public static string WeekOfTheYear()
+        {
+            return ($"Week { CultureInfo.CurrentCulture.Calendar.GetWeekOfYear(DateTime.Now, CalendarWeekRule.FirstDay, DayOfWeek.Monday)}");
+
+        }
+        #endregion
+
+        #region 01h Write a program to display day of week
+        public static int DayOfTheWeek()
+        {
+            int dayNum = 0;
+            switch (DateTime.Now.DayOfWeek)
+            {
+                case DayOfWeek.Monday:
+                    dayNum = 1;
+                    break;
+                case DayOfWeek.Tuesday:
+                    dayNum = 2;
+                    break;
+                case DayOfWeek.Wednesday:
+                    dayNum = 3;
+                    break;
+                case DayOfWeek.Thursday:
+                    dayNum = 4;
+                    break;
+                case DayOfWeek.Friday:
+                    dayNum = 5;
+                    break;
+                case DayOfWeek.Saturday:
+                    dayNum = 6;
+                    break;
+                case DayOfWeek.Sunday:
+                    dayNum = 7;
+                    break;
+                default:
+                    break;
+            }
+
+
+            return dayNum;
+        }
+
+
+
+
+
+
+
+        #endregion
+
+        #region 01j Write a program to display day of year
+        public static string IsLeapYear()
+        {
+            if (DateTime.IsLeapYear(DateTime.Now.Year))
+            {
+                return $"1j) The year {DateTime.Now.Year} is a leap year";
+            }
+            else
+            {
+                return $"1j) The year {DateTime.Now.Year} is not a leap year";
+            }
+        }
+        #endregion
+
+        #region 02 Add N years to the current date and display the new date
+        public static void DisplayModifiedYears()
+        {
+            Console.Write("Please enter the years amount to increase: ");
+            int yearsNo = int.Parse(Console.ReadLine());
+            if (yearsNo == 1)
+            {
+                Console.WriteLine($"After {yearsNo} year, the date will be: {DateTime.Now.AddYears(yearsNo)}");
+            }
+            else
+            {
+                Console.WriteLine($"After {yearsNo} years, the date will be: {DateTime.Now.AddYears(yearsNo)}");
+            }
+
+
+        }
+
+
+
+        #endregion
+
+        #region 03 Display a date in user friendly strings
+        public static void FormatDate()
+        {
```

Comment:
e ok, mai puteai sa faci folosind: ToLongDayString...sth like that si afisa dragutel